diff --git a/node_modules/admin-lte/.browserslistrc b/node_modules/admin-lte/.browserslistrc
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.bundlewatch.config.json b/node_modules/admin-lte/.bundlewatch.config.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.cache/.eslintcache b/node_modules/admin-lte/.cache/.eslintcache
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.cache/.stylelintcache b/node_modules/admin-lte/.cache/.stylelintcache
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.editorconfig b/node_modules/admin-lte/.editorconfig
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.eslintignore b/node_modules/admin-lte/.eslintignore
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.eslintrc.json b/node_modules/admin-lte/.eslintrc.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.gitattributes b/node_modules/admin-lte/.gitattributes
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.gitpod.yml b/node_modules/admin-lte/.gitpod.yml
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.stylelintignore b/node_modules/admin-lte/.stylelintignore
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/.stylelintrc.json b/node_modules/admin-lte/.stylelintrc.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/CODE_OF_CONDUCT.md b/node_modules/admin-lte/CODE_OF_CONDUCT.md
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/LICENSE b/node_modules/admin-lte/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/README.md b/node_modules/admin-lte/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/composer.json b/node_modules/admin-lte/composer.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/AdminLTEFullLogo.png b/node_modules/admin-lte/dist/assets/img/AdminLTEFullLogo.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/AdminLTELogo.png b/node_modules/admin-lte/dist/assets/img/AdminLTELogo.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/avatar.png b/node_modules/admin-lte/dist/assets/img/avatar.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/avatar2.png b/node_modules/admin-lte/dist/assets/img/avatar2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/avatar3.png b/node_modules/admin-lte/dist/assets/img/avatar3.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/avatar4.png b/node_modules/admin-lte/dist/assets/img/avatar4.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/avatar5.png b/node_modules/admin-lte/dist/assets/img/avatar5.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/boxed-bg.jpg b/node_modules/admin-lte/dist/assets/img/boxed-bg.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/boxed-bg.png b/node_modules/admin-lte/dist/assets/img/boxed-bg.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/american-express.png b/node_modules/admin-lte/dist/assets/img/credit/american-express.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/cirrus.png b/node_modules/admin-lte/dist/assets/img/credit/cirrus.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/mastercard.png b/node_modules/admin-lte/dist/assets/img/credit/mastercard.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/paypal.png b/node_modules/admin-lte/dist/assets/img/credit/paypal.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/paypal2.png b/node_modules/admin-lte/dist/assets/img/credit/paypal2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/credit/visa.png b/node_modules/admin-lte/dist/assets/img/credit/visa.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/default-150x150.png b/node_modules/admin-lte/dist/assets/img/default-150x150.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/icons.png b/node_modules/admin-lte/dist/assets/img/icons.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/photo1.png b/node_modules/admin-lte/dist/assets/img/photo1.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/photo2.png b/node_modules/admin-lte/dist/assets/img/photo2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/photo3.jpg b/node_modules/admin-lte/dist/assets/img/photo3.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/photo4.jpg b/node_modules/admin-lte/dist/assets/img/photo4.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/prod-1.jpg b/node_modules/admin-lte/dist/assets/img/prod-1.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/prod-2.jpg b/node_modules/admin-lte/dist/assets/img/prod-2.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/prod-3.jpg b/node_modules/admin-lte/dist/assets/img/prod-3.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/prod-4.jpg b/node_modules/admin-lte/dist/assets/img/prod-4.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/prod-5.jpg b/node_modules/admin-lte/dist/assets/img/prod-5.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user1-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user1-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user2-160x160.jpg b/node_modules/admin-lte/dist/assets/img/user2-160x160.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user3-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user3-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user4-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user4-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user5-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user5-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user6-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user6-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user7-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user7-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/assets/img/user8-128x128.jpg b/node_modules/admin-lte/dist/assets/img/user8-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.css b/node_modules/admin-lte/dist/css/adminlte.css
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.css.map b/node_modules/admin-lte/dist/css/adminlte.css.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.min.css b/node_modules/admin-lte/dist/css/adminlte.min.css
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.min.css.map b/node_modules/admin-lte/dist/css/adminlte.min.css.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.rtl.css b/node_modules/admin-lte/dist/css/adminlte.rtl.css
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.rtl.css.map b/node_modules/admin-lte/dist/css/adminlte.rtl.css.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.rtl.min.css b/node_modules/admin-lte/dist/css/adminlte.rtl.min.css
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/css/adminlte.rtl.min.css.map b/node_modules/admin-lte/dist/css/adminlte.rtl.min.css.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/js/adminlte.js b/node_modules/admin-lte/dist/js/adminlte.js
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/js/adminlte.js.map b/node_modules/admin-lte/dist/js/adminlte.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/js/adminlte.min.js b/node_modules/admin-lte/dist/js/adminlte.min.js
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/js/adminlte.min.js.map b/node_modules/admin-lte/dist/js/adminlte.min.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/UI/general.html b/node_modules/admin-lte/dist/pages/UI/general.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/UI/timeline.html b/node_modules/admin-lte/dist/pages/UI/timeline.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/browser-support.html b/node_modules/admin-lte/dist/pages/docs/browser-support.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/color-mode.html b/node_modules/admin-lte/dist/pages/docs/color-mode.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/components/main-header.html b/node_modules/admin-lte/dist/pages/docs/components/main-header.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/components/main-sidebar.html b/node_modules/admin-lte/dist/pages/docs/components/main-sidebar.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/faq.html b/node_modules/admin-lte/dist/pages/docs/faq.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/how-to-contribute.html b/node_modules/admin-lte/dist/pages/docs/how-to-contribute.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/introduction.html b/node_modules/admin-lte/dist/pages/docs/introduction.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/javascript/treeview.html b/node_modules/admin-lte/dist/pages/docs/javascript/treeview.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/layout.html b/node_modules/admin-lte/dist/pages/docs/layout.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/docs/license.html b/node_modules/admin-lte/dist/pages/docs/license.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/examples/lockscreen.html b/node_modules/admin-lte/dist/pages/examples/lockscreen.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/examples/login-v2.html b/node_modules/admin-lte/dist/pages/examples/login-v2.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/examples/login.html b/node_modules/admin-lte/dist/pages/examples/login.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/examples/register-v2.html b/node_modules/admin-lte/dist/pages/examples/register-v2.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/examples/register.html b/node_modules/admin-lte/dist/pages/examples/register.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/forms/general.html b/node_modules/admin-lte/dist/pages/forms/general.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/index.html b/node_modules/admin-lte/dist/pages/index.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/index2.html b/node_modules/admin-lte/dist/pages/index2.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/index3.html b/node_modules/admin-lte/dist/pages/index3.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/collapsed-sidebar.html b/node_modules/admin-lte/dist/pages/layout/collapsed-sidebar.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/fixed-complete.html b/node_modules/admin-lte/dist/pages/layout/fixed-complete.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/fixed-sidebar.html b/node_modules/admin-lte/dist/pages/layout/fixed-sidebar.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/layout-rtl.html b/node_modules/admin-lte/dist/pages/layout/layout-rtl.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/logo-switch.html b/node_modules/admin-lte/dist/pages/layout/logo-switch.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/sidebar-mini.html b/node_modules/admin-lte/dist/pages/layout/sidebar-mini.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/layout/unfixed-sidebar.html b/node_modules/admin-lte/dist/pages/layout/unfixed-sidebar.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/tables/simple.html b/node_modules/admin-lte/dist/pages/tables/simple.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/widgets/cards.html b/node_modules/admin-lte/dist/pages/widgets/cards.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/widgets/info-box.html b/node_modules/admin-lte/dist/pages/widgets/info-box.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/dist/pages/widgets/small-box.html b/node_modules/admin-lte/dist/pages/widgets/small-box.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/index.html b/node_modules/admin-lte/index.html
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/AdminLTEFullLogo.png b/node_modules/admin-lte/src/assets/img/AdminLTEFullLogo.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/AdminLTELogo.png b/node_modules/admin-lte/src/assets/img/AdminLTELogo.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/avatar.png b/node_modules/admin-lte/src/assets/img/avatar.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/avatar2.png b/node_modules/admin-lte/src/assets/img/avatar2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/avatar3.png b/node_modules/admin-lte/src/assets/img/avatar3.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/avatar4.png b/node_modules/admin-lte/src/assets/img/avatar4.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/avatar5.png b/node_modules/admin-lte/src/assets/img/avatar5.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/boxed-bg.jpg b/node_modules/admin-lte/src/assets/img/boxed-bg.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/boxed-bg.png b/node_modules/admin-lte/src/assets/img/boxed-bg.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/american-express.png b/node_modules/admin-lte/src/assets/img/credit/american-express.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/cirrus.png b/node_modules/admin-lte/src/assets/img/credit/cirrus.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/mastercard.png b/node_modules/admin-lte/src/assets/img/credit/mastercard.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/paypal.png b/node_modules/admin-lte/src/assets/img/credit/paypal.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/paypal2.png b/node_modules/admin-lte/src/assets/img/credit/paypal2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/credit/visa.png b/node_modules/admin-lte/src/assets/img/credit/visa.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/default-150x150.png b/node_modules/admin-lte/src/assets/img/default-150x150.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/icons.png b/node_modules/admin-lte/src/assets/img/icons.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/photo1.png b/node_modules/admin-lte/src/assets/img/photo1.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/photo2.png b/node_modules/admin-lte/src/assets/img/photo2.png
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/photo3.jpg b/node_modules/admin-lte/src/assets/img/photo3.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/photo4.jpg b/node_modules/admin-lte/src/assets/img/photo4.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/prod-1.jpg b/node_modules/admin-lte/src/assets/img/prod-1.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/prod-2.jpg b/node_modules/admin-lte/src/assets/img/prod-2.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/prod-3.jpg b/node_modules/admin-lte/src/assets/img/prod-3.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/prod-4.jpg b/node_modules/admin-lte/src/assets/img/prod-4.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/prod-5.jpg b/node_modules/admin-lte/src/assets/img/prod-5.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user1-128x128.jpg b/node_modules/admin-lte/src/assets/img/user1-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user2-160x160.jpg b/node_modules/admin-lte/src/assets/img/user2-160x160.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user3-128x128.jpg b/node_modules/admin-lte/src/assets/img/user3-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user4-128x128.jpg b/node_modules/admin-lte/src/assets/img/user4-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user5-128x128.jpg b/node_modules/admin-lte/src/assets/img/user5-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user6-128x128.jpg b/node_modules/admin-lte/src/assets/img/user6-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user7-128x128.jpg b/node_modules/admin-lte/src/assets/img/user7-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/assets/img/user8-128x128.jpg b/node_modules/admin-lte/src/assets/img/user8-128x128.jpg
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/config/assets.config.mjs b/node_modules/admin-lte/src/config/assets.config.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/config/astro.config.mjs b/node_modules/admin-lte/src/config/astro.config.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/config/postcss.config.mjs b/node_modules/admin-lte/src/config/postcss.config.mjs
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/config/rollup.config.js b/node_modules/admin-lte/src/config/rollup.config.js
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/.eslintrc.json b/node_modules/admin-lte/src/html/.eslintrc.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/.prettierrc.js b/node_modules/admin-lte/src/html/.prettierrc.js
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/.tsconfig.json b/node_modules/admin-lte/src/html/.tsconfig.json
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/_head.astro b/node_modules/admin-lte/src/html/components/_head.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/_scripts.astro b/node_modules/admin-lte/src/html/components/_scripts.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/dashboard/_footer.astro b/node_modules/admin-lte/src/html/components/dashboard/_footer.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/dashboard/_sidenav.astro b/node_modules/admin-lte/src/html/components/dashboard/_sidenav.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/dashboard/_topbar.astro b/node_modules/admin-lte/src/html/components/dashboard/_topbar.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/browser-support.mdx b/node_modules/admin-lte/src/html/components/docs/browser-support.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/color-mode.mdx b/node_modules/admin-lte/src/html/components/docs/color-mode.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/components/main-header.mdx b/node_modules/admin-lte/src/html/components/docs/components/main-header.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/components/main-sidebar.mdx b/node_modules/admin-lte/src/html/components/docs/components/main-sidebar.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/faq.mdx b/node_modules/admin-lte/src/html/components/docs/faq.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/how-to-contribute.mdx b/node_modules/admin-lte/src/html/components/docs/how-to-contribute.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/introduction.mdx b/node_modules/admin-lte/src/html/components/docs/introduction.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/docs/license.mdx b/node_modules/admin-lte/src/html/components/docs/license.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/components/javascript/treeview.mdx b/node_modules/admin-lte/src/html/components/javascript/treeview.mdx
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/env.d.ts b/node_modules/admin-lte/src/html/env.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/UI/general.astro b/node_modules/admin-lte/src/html/pages/UI/general.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/UI/timeline.astro b/node_modules/admin-lte/src/html/pages/UI/timeline.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/browser-support.astro b/node_modules/admin-lte/src/html/pages/docs/browser-support.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/color-mode.astro b/node_modules/admin-lte/src/html/pages/docs/color-mode.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/components/main-header.astro b/node_modules/admin-lte/src/html/pages/docs/components/main-header.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/components/main-sidebar.astro b/node_modules/admin-lte/src/html/pages/docs/components/main-sidebar.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/faq.astro b/node_modules/admin-lte/src/html/pages/docs/faq.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/how-to-contribute.astro b/node_modules/admin-lte/src/html/pages/docs/how-to-contribute.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/introduction.astro b/node_modules/admin-lte/src/html/pages/docs/introduction.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/javascript/treeview.astro b/node_modules/admin-lte/src/html/pages/docs/javascript/treeview.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/layout.astro b/node_modules/admin-lte/src/html/pages/docs/layout.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/docs/license.astro b/node_modules/admin-lte/src/html/pages/docs/license.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/examples/lockscreen.astro b/node_modules/admin-lte/src/html/pages/examples/lockscreen.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/examples/login-v2.astro b/node_modules/admin-lte/src/html/pages/examples/login-v2.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/examples/login.astro b/node_modules/admin-lte/src/html/pages/examples/login.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/examples/register-v2.astro b/node_modules/admin-lte/src/html/pages/examples/register-v2.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/examples/register.astro b/node_modules/admin-lte/src/html/pages/examples/register.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/forms/general.astro b/node_modules/admin-lte/src/html/pages/forms/general.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/index.astro b/node_modules/admin-lte/src/html/pages/index.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/index2.astro b/node_modules/admin-lte/src/html/pages/index2.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/index3.astro b/node_modules/admin-lte/src/html/pages/index3.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/collapsed-sidebar.astro b/node_modules/admin-lte/src/html/pages/layout/collapsed-sidebar.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/fixed-complete.astro b/node_modules/admin-lte/src/html/pages/layout/fixed-complete.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/fixed-sidebar.astro b/node_modules/admin-lte/src/html/pages/layout/fixed-sidebar.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/layout-rtl.astro b/node_modules/admin-lte/src/html/pages/layout/layout-rtl.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/logo-switch.astro b/node_modules/admin-lte/src/html/pages/layout/logo-switch.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/sidebar-mini.astro b/node_modules/admin-lte/src/html/pages/layout/sidebar-mini.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/layout/unfixed-sidebar.astro b/node_modules/admin-lte/src/html/pages/layout/unfixed-sidebar.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/tables/simple.astro b/node_modules/admin-lte/src/html/pages/tables/simple.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/widgets/cards.astro b/node_modules/admin-lte/src/html/pages/widgets/cards.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/widgets/info-box.astro b/node_modules/admin-lte/src/html/pages/widgets/info-box.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/html/pages/widgets/small-box.astro b/node_modules/admin-lte/src/html/pages/widgets/small-box.astro
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_app-content.scss b/node_modules/admin-lte/src/scss/_app-content.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_app-footer.scss b/node_modules/admin-lte/src/scss/_app-footer.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_app-header.scss b/node_modules/admin-lte/src/scss/_app-header.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_app-main.scss b/node_modules/admin-lte/src/scss/_app-main.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_app-sidebar.scss b/node_modules/admin-lte/src/scss/_app-sidebar.scss
old mode 100644
new mode 100755
index 9bf7e2b..e736da0
--- a/node_modules/admin-lte/src/scss/_app-sidebar.scss
+++ b/node_modules/admin-lte/src/scss/_app-sidebar.scss
@@ -582,7 +582,6 @@ body:not(.app-loaded) {
 
 @if $enable-dark-mode {
   @include color-mode(dark) {
-    &.app-sidebar,
     .app-sidebar {
       --#{$lte-prefix}sidebar-hover-bg: #{$lte-sidebar-hover-bg-dark};
       --#{$lte-prefix}sidebar-color: #{$lte-sidebar-color-dark};
diff --git a/node_modules/admin-lte/src/scss/_app-wrapper.scss b/node_modules/admin-lte/src/scss/_app-wrapper.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_bootstrap-variables.scss b/node_modules/admin-lte/src/scss/_bootstrap-variables.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_callouts.scss b/node_modules/admin-lte/src/scss/_callouts.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_cards.scss b/node_modules/admin-lte/src/scss/_cards.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_compact-mode.scss b/node_modules/admin-lte/src/scss/_compact-mode.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_direct-chat.scss b/node_modules/admin-lte/src/scss/_direct-chat.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_docs.scss b/node_modules/admin-lte/src/scss/_docs.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_dropdown.scss b/node_modules/admin-lte/src/scss/_dropdown.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_info-box.scss b/node_modules/admin-lte/src/scss/_info-box.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_miscellaneous.scss b/node_modules/admin-lte/src/scss/_miscellaneous.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_mixins.scss b/node_modules/admin-lte/src/scss/_mixins.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_progress-bars.scss b/node_modules/admin-lte/src/scss/_progress-bars.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_root.scss b/node_modules/admin-lte/src/scss/_root.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_small-box.scss b/node_modules/admin-lte/src/scss/_small-box.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_table.scss b/node_modules/admin-lte/src/scss/_table.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_timeline.scss b/node_modules/admin-lte/src/scss/_timeline.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_toasts.scss b/node_modules/admin-lte/src/scss/_toasts.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_variables-dark.scss b/node_modules/admin-lte/src/scss/_variables-dark.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/_variables.scss b/node_modules/admin-lte/src/scss/_variables.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/adminlte.scss b/node_modules/admin-lte/src/scss/adminlte.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/mixins/_animations.scss b/node_modules/admin-lte/src/scss/mixins/_animations.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/mixins/_scrollbar.scss b/node_modules/admin-lte/src/scss/mixins/_scrollbar.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/pages/_lockscreen.scss b/node_modules/admin-lte/src/scss/pages/_lockscreen.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/pages/_login_and_register.scss b/node_modules/admin-lte/src/scss/pages/_login_and_register.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/parts/_components.scss b/node_modules/admin-lte/src/scss/parts/_components.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/parts/_core.scss b/node_modules/admin-lte/src/scss/parts/_core.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/parts/_extra-components.scss b/node_modules/admin-lte/src/scss/parts/_extra-components.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/parts/_miscellaneous.scss b/node_modules/admin-lte/src/scss/parts/_miscellaneous.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/scss/parts/_pages.scss b/node_modules/admin-lte/src/scss/parts/_pages.scss
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/adminlte.ts b/node_modules/admin-lte/src/ts/adminlte.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/card-widget.ts b/node_modules/admin-lte/src/ts/card-widget.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/direct-chat.ts b/node_modules/admin-lte/src/ts/direct-chat.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/fullscreen.ts b/node_modules/admin-lte/src/ts/fullscreen.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/layout.ts b/node_modules/admin-lte/src/ts/layout.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/push-menu.ts b/node_modules/admin-lte/src/ts/push-menu.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/treeview.ts b/node_modules/admin-lte/src/ts/treeview.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/ts/util/index.ts b/node_modules/admin-lte/src/ts/util/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/src/utils/index.js b/node_modules/admin-lte/src/utils/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/admin-lte/tsconfig.json b/node_modules/admin-lte/tsconfig.json
old mode 100644
new mode 100755
